### PR TITLE
travis: try npm test twice

### PR DESCRIPTION
**Pull request by fippo**
band-aid for flaky tests. I am looking at you DTMF...

### CODE REVIEW

**Comment by dagingaa** [comment]
Ironic that the test fails... :P If I were Brad now I would tell you to not put a bandaid on it and see what the root cause is and if it can be fixed. If the DTMF tests are flaky should we disable them instead for the time being?

**Comment by fippo** [comment]
the dtmf test needs an additional check. Pondering whether to actually remove it... it has been flaky for ages.

**Comment by fippo** [comment]
Looking at the actual test showed that the anti-flake method was only used in one of the two. Updated... also stop sub-commenting about @bradleythughes :-p

**Comment by dagingaa** [comment]
I couldn’t autocomplete his name for some reason :p

File: .travis.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ before_script:
 
 script:
   - node_modules/.bin/grunt
-  - npm test
+  - npm test || npm test
```
Comment (dagingaa) [inline .travis.yml:43]: Do we still need this then?

**Review by dagingaa — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (fippo) [reply .travis.yml:43]: i can make sure all the emails go to you... lets keep it :-p

Comment (fippo) [reply .travis.yml:43]: (there are also some firefox timeout crash things....)

**Pull request merged**